app/scripts/lib/segment/analytics.js

```diff
@@ -0,0 +1,212 @@
+import removeSlash from 'remove-trailing-slash';
```

Comment:
for some reason, I'm not seeing events after setting up `.metamaskrc` variables, switching the "Participate in MetaMetrics" setting on, turning off blockers, and then interacting with our app